docs/reference/release-notes/highlights-6.5.0.asciidoc

```diff
@@ -43,3 +43,13 @@ analysis. A new `multi_bucket_impact` property in the
 analysis influences the score. In {kib}, anomalies with medium or high 
 multi-bucket impact are depicted in the *Anomaly Explorer* and the 
 *Single Metric Viewer* with a cross symbol instead of a dot. 
+
+[float]
+=== Monitor {es} with {metricbeat} (beta)
+
+In 6.4 and later, you can use {metricbeat} to collect data about {kib} and ship 
```

Comment:
Since these are ES release highlights and this item is about monitoring ES with MB, should we mention Kibana, LS, and Beats monitoring in here as well?

Reply:
I think since ES has a role in routing monitoring data from other products, it's worthwhile to mention those products here too.